.github/workflows/release.yaml

```diff
@@ -79,6 +79,23 @@ jobs:
           GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           ORG_GITHUB_TOKEN: ${{ secrets.ORG_GITHUB_TOKEN }}
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository: OpsLevel/homebrew-tap
+          token: ${{ secrets.ORG_GITHUB_TOKEN }}
+          path: homebrew-tap
+      - name: Update homebrew
+        working-directory: homebrew-tap
+        run: |
+          cd Formula
+          awk -v ver="${RELEASE_VERSION}" '/version/, /tags/ {sub(/[0-9]+\.[0-9]+\.[0-9]+/, ver)} {print}' kubectl.rb > tmp.rb
```

Comment:
awk breakdown: modifies [kubectl.rb](https://github.com/OpsLevel/homebrew-tap/blob/main/Formula/kubectl.rb) 

awk 
`-v ver="${RELEASE_VERSION}"` // set `ver` as variable equal to ${RELEASE_VERSION}
`'/version/, /tags/` // start on line with "version", stop on  line with "tags"
`{sub(/[0-9]+\.[0-9]+\.[0-9]+/, ver)}` // substitute regex pattern for version numbers with `ver` variable
`{print}' kubectl.rb` // print file with substitutions to kubectl.rb

Reply:
Not that i don't love the ingenuity but its the complexity and possible brittleness i'm worried about.

If we are going to rip homebrew out of go releaser maybe we should also look for a formula building replacement.

It looks like brew has some stuff built in and there are a number of github actions out there to bump the version of formula in a tap.

``` brew bump-formula-pr --help```

IIRC brew comes pre-installed on the macos github actions runners and it is easily installable on linux.

Reply:
It makes sense _something_ would be out there for this 😄 
[link for later](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md)

Reply:
Reading more deeply it seems `brew bump-formula-pr` is used for updating homebrew core taps and it doesn't seem to work with tap's (at least not that I could understand)